app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1831,8 +1831,6 @@ class BrowserTabFragment :
                 is DaxBubbleCta -> showDaxCta(configuration)
                 is DialogCta -> showDaxDialogCta(configuration)
             }
-
-            viewModel.onCtaShown()
```

Comment:
We were calling this method without really knowing if a CTA was being shown or not. I've removed it from here and added it to the specific places after showing the CTA.